build-scripts/rollup.browser.config.js

```diff
@@ -38,6 +38,7 @@ export default function createBrowserConfig(options = {}, targets) {
           plugins: [
             '@babel/plugin-external-helpers',
             ['@babel/plugin-transform-runtime', { corejs: 2 }],
+            ['@babel/plugin-proposal-class-properties', { loose: true }],
```

Comment:
This allows us to use class properties. I'll start migrating over to use them in another PR and include better inline documentation.